src/core/apollo-client.js

```diff
@@ -0,0 +1,7 @@
+import ApolloClient from "apollo-boost";
+
+const client = new ApolloClient({
+    uri: `https://mj97l35pk9.lp.gql.zone/graphql`
```

Comment:
no need for `template literals` here - just ordinary string

Reply:
Sure